### ISSUE TITLE
Featured image not being updated

### ISSUE THREAD

**Issue by hvelarde** [collective/collective.behavior.featuredimage#21]
Steps to reproduce:
- install the package and don't configure it
- enable the behavior on one content type
- create a new item and add information for a featured image
- an image with and error will be generated:

![featuredimage](https://cloud.githubusercontent.com/assets/341393/13093379/13aa3a00-d4ec-11e5-8e2a-8243d6da6d63.png)
- upload a background image via the configlet
- edit the item

The featured image is not updated.

**Comment by rodfersou** [comment]
@hvelarde  I think I fixed this bug [here](https://github.com/collective/collective.behavior.featuredimage/commit/dea8e8db11417b999e00c9bc1041d44c43013f50).. could you please confirm that?

Note that you must publish the item to generate the featured image.

**Comment by hvelarde** [comment]
@rodfersou can you remind me why we're updating only on workflow change?

**Comment by rodfersou** [comment]
@hvelarde because phantomjs can't generate screenshot in the first time we create the object (the view is available, but css is not, resulting a broken image).

This don't affect the preview.

**Comment by hvelarde** [comment]
I think we must solve that issue then.

**Comment by rodfersou** [comment]
Oh.. I understand wrong the issue.. when you update the controlpanel image should update all featured images, right?

**Comment by hvelarde** [comment]
no, it makes no sense as Facebook [caches the image forever](https://developers.facebook.com/docs/sharing/webmasters):

> Images are cached based on the URL and won't be updated unless the URL changes.

**Comment by rodfersou** [comment]
right.. then I think this problem is invalid, can you confirm that please?

**Comment by hvelarde** [comment]
no, is not; I must be able to modify the image before publishing it; check again the steps to reproduce it; also, we will need to figure out a way to change the image URL if we edit the quote or author.

**Comment by rodfersou** [comment]
why we need to modify the generated image when it is still not in use?

again, the preview work fine, and we can see how the final image will be generated when you publish the featured image.

**Comment by rodfersou** [comment]
well sorry.. I'll try to reproduce again the steps, but I'm afraid the problem is related with your browser cache.
